Chapter_5/sagemaker/README.md

```diff
@@ -1,5 +1,7 @@
-#### Sagemaker
+## Utilizing SageMaker for ETL
 
-- Introduction to **Sagemaker Studio** with a document [here](./sagemaker_intro.md).
-- [Example Jupyter Notebook](./cifar10-tensorflow.ipynb) using [ResNet](https://arxiv.org/abs/1512.03385) with [CIFAR 10](https://www.cs.toronto.edu/~kriz/cifar.html).
-  This code has been revised after being taken from the AWS sample [here](https://aws.amazon.com/getting-started/hands-on/train-tune-deep-learning-model-amazon-sagemaker/) in order to run successfully.
+[SageMaker](https://aws.amazon.com/sagemaker/) is an end-to-end service for machine learning. You can configure SageMaker to handle data processing, model development with notebooks, model training, as well as deployment of models to a production setting.
+
+In this repository,
+* we explain [how to set up a SageMaker Studio](./sagemaker_studio.md), a web-based development environment for SageMaker.
+* [A sample model training implementation on SageMaker](./cifar10-tensorflow.ipynb). The model is [ResNet](https://arxiv.org/abs/1512.03385) trained with [CIFAR 10](https://www.cs.toronto.edu/~kriz/cifar.html) for image classification.
```

Comment:
@leninworld  the notebook looks unpolished (has error message as well)

Can you make sure the code is cleaned up and runnable?

Reply:
The last few blocks are supposed to be removed (and forgotten) and are now removed. The rest of the code was already tested 3 times for cifar and should be fine. good.